webapi/src/main/scala/org/knora/webapi/responders/v2/ontology/Cache.scala

```diff
@@ -85,7 +83,6 @@ object Cache extends LazyLogging {
     superClassOfRelations: Map[SmartIri, Set[SmartIri]],
     subPropertyOfRelations: Map[SmartIri, Set[SmartIri]],
     superPropertyOfRelations: Map[SmartIri, Set[SmartIri]],
-    guiAttributeDefinitions: Map[SmartIri, Set[SalsahGuiAttributeDefinition]],
```

Comment:
why can this be removed from the cache entirely? I mean... nice that it can... but why?

Reply:
Because, as far as I've understood, it was not used in the cache. The `guiAttributeDefinitions` were read from the salsah gui ontology but there was no validation done with it.